dnf/cli/commands/shell.py

```diff
@@ -115,6 +115,8 @@ def onecmd(self, line):
                     cmd.run()
                 except Exception as e:
                     logger.error(_("Error:") + " " + e.value)
+                except:
```

Comment:
The exception (from argparser) wasn't caught. Another "except" was needed.
But there was another bug. Class "Exception" have not attribute "value". "dnf.exceptions.Error" must be used.
I made fixup.